code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -238,6 +238,30 @@
 		beakers += B2
 		icon_state = initial(icon_state) +"_locked"
 
+/obj/item/weapon/grenade/chem_grenade/gas
+	name = "NT. 53 'Sandman' grenade"
+	desc = "Used for pacifying rooms full of people with less-than-lethal force. It warns against use on people with lung defects."
+	path = 1
+	stage = 2
+
+	/obj/item/weapon/grenade/chem_grenade/gas/New()

```

Comment:
We talked about this. Tab this line, and all the lines up-to-and-including 263 back one space.